content/integrations/google_container_engine.md

```diff
@@ -5,6 +5,7 @@ kind: integration
 doclevel: basic
 newhlevel: true
 git_integration_title: google_container_engine
+description: "Monitor your GCE containers' resource usage."
```

Comment:
Google Container Engine and/or GKE.  Not "GKE Containers"

Reply:
👍